File: frontend/src/data/sample.js
import { floodFrames } from './prototype.js'
export const simulationId = '44444444-4444-4444-8444-444444444444'
export const sampleTimestamp = '2026-09-05T06:01:00Z'
export const sampleSummary = { flooded_area_km2: 42.8, maximum_depth_m: 8.4, population_exposed: 18420, buildings_affected: 3260, critical_assets: 27, is_sample: true, updated_at: sampleTimestamp }
export const sampleTimeline = floodFrames.map(frame => ({ minute:frame.minute, depth_m:frame.depth, flooded_area_km2:frame.area }))
export const riverPath = [83.865,21.535, 83.881,21.509, 83.908,21.484, 83.939,21.469, 83.962,21.445, 83.982,21.418, 84.022,21.396, 84.068,21.374, 84.102,21.348]

File: backend/app/services/sample.py
from uuid import UUID

DAM_ID = UUID("11111111-1111-4111-8111-111111111111")
PARTIAL_ID = UUID("22222222-2222-4222-8222-222222222222")
MAJOR_ID = UUID("33333333-3333-4333-8333-333333333333")
SIMULATION_ID = UUID("44444444-4444-4444-8444-444444444444")
SUMMARY_ID = UUID("55555555-5555-4555-8555-555555555555")
TIMELINE = [
    {"minute": 0, "depth_m": 0, "flooded_area_km2": 0},
    {"minute": 15, "depth_m": 2.1, "flooded_area_km2": 9.6},
    {"minute": 30, "depth_m": 5.3, "flooded_area_km2": 24.2},
    {"minute": 60, "depth_m": 8.4, "flooded_area_km2": 42.8},
]
